capstone/client/src/components/StoreDetailCards.js

```diff
@@ -15,43 +15,47 @@
  */
 
 import React, { Component } from 'react';
-import { Grid, Typography, List, ListItem, ListItemText } from '@material-ui/core';
-import { Map } from 'google-maps-react';
+import { GoogleApiWrapper, Map } from 'google-maps-react';
+import { Card, Grid, List, ListItem, ListItemText, Typography } from '@material-ui/core';
+
+import APIKey from './APIKey.js';
 import './styles.css';
 
 class StoreDetailCards extends Component {
-
-    render() {
-        const storeDetailCards = this.props.stores.map((store) => (
-            <div>
-              <Typography variant='h4'>{store.storeName}</Typography>
-              <Typography variant='h6'>Address: {store.address}</Typography>
-              <Grid container alignItems="stretch">
-                <Grid xs>
-                  <Map id="google-map" google={this.props.google} zoom={14}>
-                  </Map>
-                </Grid>
-                <Grid xs>
-                  <Typography variant='subtitle1'>Has:</Typography>
-                  <List>
-                  {Object.keys(store.items).map((itemType, item) => (
-                    <ListItem>
-                      <ListItemText>
-                        {item.itemName} (${item.itemPrice})
-                      </ListItemText>
-                    </ListItem>
-                  ))}
-                  </List>
-                </Grid>
-              </Grid>
-            </div>
-          ));
-        return (
-            <div>
-              {storeDetailCards[0]}
-            </div>
-        )
-    }
+  render() {
+    const storeDetailCards = this.props.stores.map((store) => (
+      <div>
+        <Typography variant='h4'>{store.storeName}</Typography>
+        <Typography variant='h6'>Address: {store.address}</Typography>
+        <Grid container alignItems="stretch">
+          <Grid item component={Card} xs>
+            <Map id="google-map" google={this.props.google} zoom={14} style={{width: '25%', height: '25%'}}>
```

Comment:
Move the Style to CSS file

Reply:
The styles.css file is really weird. In pages that don't import it (like ListPage.js), the style applies, but regardless of whether I import it or not in this file, Google Maps style doesn't change unless I put the style changes in the file (StoreDetailCards.js) itself. For now, I put the json style in a separate variable in the render function, until we can look at it later?